### ISSUE TITLE
Various suggestions

### ISSUE THREAD

**Issue by ChatFawkes** [Elecs-Mods/GregsProspecting-II#1]
I had a few suggestion that I'd thought be nice so I'll leave them in this issue:
~~\- Support for mod ores~~
~~\- Ability to increase range via a config option~~
(both are implemented now)

Also not sure if a bug but currently I'm trying to fly over oceans and find veins but everything just appears as red on the color radar and yellow for the regular one. Would be nice if that could be disabled, so you can easily find veins while flying over oceans.

**Comment by Elec332** [comment]
The display shows the hardness, it just colours to the colour of water, ill see if I can make a config option/hotkey for that aswell

**Comment by ChatFawkes** [comment]
Ahh okay, that makes sense now because lots of ores and limestone were showing up as brown (assuming that's iron level hardneess).

**Comment by Elec332** [comment]
Yes, the item checks for the hardness of blocks, not what block it is ;)
(So mod ores is already in :P)

**Comment by ChatFawkes** [comment]
If it isn't too hard: Would it be possible for you to implement a way to blacklist certain blocks from showing up in the Mining Radar?

**Comment by Elec332** [comment]
Ill look into that ;)
